src/main/java/com/czertainly/core/util/CertificateRequestUtils.java

```diff
@@ -141,4 +145,16 @@ public static String normalizeCsrContent(String csr) {
         return csr;
     }
 
+    public static CertificateRequest createCertificateRequest(byte[] certificateRequest, CertificateRequestFormat format) throws IOException {
```

Comment:
We should use custom `CertificateRequestException` to associate exceptions and messages related to implementation of requests.